doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -133,6 +141,7 @@ CPU reservation for host: 1 core per NUMA
     - minimum: two nodes per profile
 -	HW specifications
     -	Boot disks are dedicated with Flash technology disks
+    - Additional 4 cores overhead per numa in case of DPDK usage for Network Intensive nodes
```

Comment:
@samuelhellec do we need 4 physical cores per NUMA or 4 virtual cores per NUMA? I think it is 2 physical cores (including their threads) per NUMA.

Reply:
@pgoyal01 I mean 4 physical cores. I tried to find  a recommandation on openstack docs but the best I came up with is https://docs.openstack.org/fuel-docs/newton/userdocs/fuel-user-guide/configure-environment/nfv/nfv-configure.html which is only in neutron release

Reply:
@samuelhellec we only configure 2 physical cores (and their sibling threads) per NUMA. Do we need to specify the exact number or can we state 2 or more.

Reply:
@pgoyal01 , I investigate a bit more and we can have difference depending on the frequency of the CPU we have and the performance we want to deliver per compute. 
So I guess we will never reach an agreement on an exact value.

So I propose the following wording:
In case of DPDK usage extra cores must be reserved . the number of core to book for this overhead depend on the CPU frequency (Mpps available per core) and the number of Mpps we want to deliver by node.

Reply:
PMD = Poll Mode Driver
PHY = To manage physical NICs
VHU = To manage virtual NICs

This assumes two active and two passive NICs in a bond

Reply:
@samuelhellec The throughput Mpps depends both on the CPU frequency but more significantly on the packet size.  Performance results demonstrate that while throughput initially increases with CPU frequency it begins to fall after a certain CPU frequency. This is because DPDK does not use  interrupts,  but  utilizes  a  busy  wait  loop  that  polls  the  NIC until at least one packet is returned. Thus, at some point the fast CPU causes DPDK to poll the NIC too often affecting the throughput.

Even with a single core @3GHz is able to achieve throughput of 30+Mpps and as you know for DPDK the performance is linear with number of cores. 

My I suggest, 
In case of DPDK usage, cores (together with their sibling threads) must be reserved for DPDK Poll Mode Drivers (PMD), hugepages set to at least 1 GB, and  all power settings disabled in the BIOS. The number of cores reserved for this overhead depends on the desired throughput (the number of Mpps we want to deliver).

Reply:
@pgoyal01 I am okay with your suggestion

@iangardner22 @deebhatia are you ok with this?

Reply:
The latest suggestion by @pgoyal01 is very good. We can't specify exact value.  Btw: allowed packet loss will be another factor.

Reply:
Okay with the language. One thing here is that no of cores for desired throughput is also limited by the no of queues present at the NIC.

Reply:
Last 4 lines of @pgoyal01 are very good for current VNFs in OpenStack.

Reply:
Yes happy with that - thanks @pgoyal01

Reply:
I corrected with the latest statement of pankaj where we all agree on